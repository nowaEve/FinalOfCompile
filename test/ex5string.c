int main(){

    char a = 'h';
    char c = 'e';
    char d = 'l';
    char b[10];

    // for(i = 0; i < 10; ++i){
    //     b[i] = i;
    // }

    // for(i = 0; i < 10; ++i){
    //     print b[i];
    // }
    b[0] = a;
    b[1] = c;
    b[2] = d;
    print b[1]+b[2];

} 