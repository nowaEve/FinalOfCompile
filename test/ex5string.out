24 19 0 5 25 13 27 104 12 13 0 1 1 27 101 12 13 0 2 1 27 108 12 15 10 14 0 9 2 13 0 13 1 11 13 11 12 15 -1 13 0 13 1 11 0 1 1 13 0 1 1 11 12 15 -1 13 0 13 1 11 0 2 1 13 0 2 1 11 12 15 -1 13 0 13 1 11 0 1 1 11 13 0 13 1 11 0 2 1 11 1 22 21 14